localSetup/projects/nginx/nginx-conf.d/default.conf

```diff
@@ -0,0 +1,10 @@
+server {
+  listen        80 default_server;
```

Comment:
The web server was missing a default vhost, so added one to avoid sudden change in healthcheck behaviour.